src/components/Patient/PatientDetailsTab/ShiftingHistory.tsx

```diff
@@ -43,31 +47,73 @@ const ShiftingHistory = (props: PatientProps) => {
     },
   );
 
+  const CURRENT_STATUSES = [
+    "APPROVED",
+    "DESTINATION APPROVED",
+    "PATIENT TO BE PICKED UP",
+    "TRANSFER IN PROGRESS",
+  ];
+  const PREVIOUS_STATUSES = ["COMPLETED", "PATIENT EXPIRED", "CANCELLED"];
+
+  const filteredShiftData = shiftData?.results
+    ? activeTab === "current"
+      ? shiftData.results.filter((shift) =>
+          CURRENT_STATUSES.includes(shift.status),
+        )
+      : shiftData.results.filter((shift) =>
+          PREVIOUS_STATUSES.includes(shift.status),
+        )
+    : [];
   return (
     <section className="mt-4">
       <div className="flex justify-between items-center">
-        <h2 className="my-4 ml-0 text-2xl font-semibold leading-tight">
-          {t("shifting_history")}
-        </h2>
-        <ButtonV2
-          className=""
+        <div className="flex mb-4 bg-gray-100 w-fit rounded-lg px-2 py-1">
```

Comment:
Adjust the font sizes of the tabs' Play around with it - match font size of shift patient button perhaps.